vcd/vdc_group_common_test.go

```diff
@@ -40,19 +40,20 @@ const testAccVcdVdcGroupNew = `
     default = true
   }
 
+  network_quota = 100
+
   enabled                    = true
   enable_thin_provisioning   = true
   enable_fast_provisioning   = true
   delete_force               = true
   delete_recursive           = true
-  elasticity      			 = true
+  elasticity      			     = true
```

Comment:
This is probably just how GitHub renders it, but just in case (it shows indentation as off):

<img width="515" alt="Screenshot 2022-03-22 at 15 23 10" src="https://user-images.githubusercontent.com/11330689/159491592-589526e3-b14a-4532-9937-fe94b2e94320.png">

Reply:
This could have been some tabs/vs spaces thing. Tried to indent from scratch